terraform/development/shared/main.tf

```diff
@@ -153,10 +153,10 @@ module "certs_for_new_domain" {
     aws.us-east-1 = aws.us-east-1
   }
 
-  cloudfront_domain_name         = local.new_app_host
-  cloudfront_additional_names    = [local.app_host]
-  load_balancer_domain_name      = local.new_load_balancer_domain_name
-  load_balancer_additional_names = [local.load_balancer_domain_name]
+  cloudfront_domain_name         = local.app_host
+  cloudfront_additional_names    = [local.old_app_host]
+  load_balancer_domain_name      = local.load_balancer_domain_name
+  load_balancer_additional_names = [local.old_load_balancer_domain_name]

```

Comment:
Because of the change above, this stays the same and the `certificates` module above will remove the old domain certs it previously managed and create ones for the new domain only (instead of the dual certs covering both).